src/Native/Windows/build-native.cmd

```diff
@@ -9,7 +9,7 @@ set __CMakeBinDir=""
 set __IntermediatesDir=""
 set __BuildArch=AnyCPU
 set __VCBuildArch=x86_amd64
-set CMAKE_BUILD_TYPE=Debug
+set CMAKE_BUILD_TYPE=Release
```

Comment:
I see.  Why change the default?  Don't we normally default to debug unless release is explicitly chosen?